plugins/uefi-capsule/fwupdate.md

```diff
@@ -0,0 +1,37 @@
+% fwupdate(1) @PACKAGE_VERSION@ | fwupdate man page
+
+NAME
+====
+
+**fwupdate** — debugging utility for UEFI firmware updates
+
+SYNOPSIS
+========
+
+| **fwupdate** \[CMD]
+
+DESCRIPTION
+===========
+
+**fwupdate** is a deprecated tool that allows deploying capsule updates.
+
+OPTIONS
+=======
+
+The fwupdate command takes various options depending on the action.
+Run **fwupdate --help** for the full list.
```

Comment:
This is the part that I don't like about having documentation that isn't automatically built.  You either need to always remember to keep it in sync or you end up with stuff like this.

Reply:
Yup, we used to have help2man but we ripped that out as it was fragile IIRC. I don't actually see much point duplicating the --help output, I see the man page as much more helpful to see the big picture and explain how all the tools link together.

Reply:
I guess my personal opinion is that man pages are an antiquated difficult to use documentation mechanism.

I would personally rather be pointed at a URL to some colorful rich documentation that I can jump back and forth from hyperlinks as I encounter related information.

Reply:
> man pages are an antiquated difficult to use documentation mechanism

Yup, I don't disagree -- but I don't think the manpages are aimed at either of us. Unfortunately in RHEL we have mandatory checks that they must exist -- hence the stubs we have right now. A few people expect to use them -- especially people coming from doing low-level server admin. When we tell them "don't use dbxtool, use fwupdmgr" they're the sort of person that's going to read the manpage rather than the fwupd wiki. That said, I think including URLs in the manpages is probably a good idea.

Reply:
Maybe the right balance is some light text that will age well in the man pages and the URL to more interesting information both then.